### PR TITLE
[ADD] Chinese language pack

### PR DESCRIPTION
**Pull request by niulinlnc**

### CODE REVIEW

File: formio/data/formio_translations_zh_CN.xml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        </record>
+        <record id="i18n_zh_CN_pattern" model="formio.translation">
+            <field name="lang_id" ref="base.lang_zh_CN"/>
+            <field name="source_id" ref="formio.i18n_pattern"/>
+            <field name="value">{{field}} 不匹配模式：{{pattern}}</field>
+        </record>
+        <record id="i18n_zh_CN_minLength" model="formio.translation">
+            <field name="lang_id" ref="base.lang_zh_CN"/>
+            <field name="source_id" ref="formio.i18n_minLength"/>
+            <field name="value">{{field}} 必须至少 {{length}} 字符.</field>
```
Comment (bobslee) [inline formio/data/formio_translations_zh_CN.xml:31]: I'm not Chinese, but curious, so what about ? `{{field}} 必须至少 {{length}} 字符数`

File: formio/data/formio_translations_zh_CN.xml [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        </record>
+        <record id="i18n_zh_CN_invalid_date" model="formio.translation">
+            <field name="lang_id" ref="base.lang_zh_CN"/>
+            <field name="source_id" ref="formio.i18n_invalid_date"/>
+            <field name="value">{{field}} 不是有效的日期格式.</field>
+        </record>
+        <record id="i18n_zh_CN_invalid_day" model="formio.translation">
+            <field name="lang_id" ref="base.lang_zh_CN"/>
+            <field name="source_id" ref="formio.i18n_invalid_day"/>
+            <field name="value">{{field}} 不是有效的日期格式（天）.</field>
```
Comment (bobslee) [inline formio/data/formio_translations_zh_CN.xml:81]: `{{field}} 无效的日子.` ??

**Comment by bobslee** [comment]
@niulinlnc thanks for the translations. 
The PO-file is just too extensive for me to review.
You'll recheck the small comments? Or shall I merge?

**Comment by niulinlnc** [comment]
The translation package should basically be no problem. If the translation is not appropriate, I can slowly correct it in the future.

**Pull request merged**